library/src/main/java/org/apache/apex/malhar/lib/state/spillable/SpillableMapImpl.java

```diff
@@ -236,13 +237,27 @@ public void beginWindow(long windowId)
   @Override
   public void endWindow()
   {
+    boolean isTimeUnifiedStore = (store instanceof ManagedTimeUnifiedStateImpl);
```

Comment:
The only reason to add it as class variable is due to performance, right? The benefit is increase very very little performance. It's happened every endWindow()  instead of each tuple, and the operation instanceof is in fact pretty light. But the problem is increase the complexity. I don't think buy it